### ISSUE TITLE
IE11 no longer supported when using Angular9 - Npm version?

### ISSUE THREAD

**Issue by LeahPike** [500tech/angular-tree-component#789]
# Reproduce
Please reproduce here if reporting a bug, or requesting help:
IE11 is no longer supported when using Angular 9 and the latest version of the tree component, you get the following error message now

![image](https://user-images.githubusercontent.com/6124100/73533040-d793a180-4415-11ea-846f-d7bc786784b6.png)

Here is the package.json file so you can see the references

```
{
  "name": "angular",
  "version": "0.0.0",
  "private": true,
  "dependencies": {
    "@angular/common": "^9.0.0-rc.9",
    "@angular/compiler": "^9.0.0-rc.9",
    "@angular/core": "^9.0.0-rc.9",
    "@angular/forms": "^9.0.0-rc.9",
    "@angular/platform-browser": "^9.0.0-rc.9",
    "@angular/platform-browser-dynamic": "^9.0.0-rc.9",
    "@angular/router": "^9.0.0-rc.9",
    "angular-tree-component": "^8.5.2",
    "core-js": "^2.5.7",
    "rxjs": "^6.3.3",
    "tslib": "^1.10.0",
    "zone.js": "^0.8.26"
  },
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.10.0",
    "@angular/cli": "^9.0.0-rc.9",
    "@angular/compiler-cli": "^9.0.0-rc.9",
    "@angular/language-service": "^9.0.0-rc.9",
    "@types/node": "~8.9.4",
    "@types/jasmine": "~2.8.8",
    "@types/jasminewd2": "~2.0.3",
    "codelyzer": "~4.5.0",
    "jasmine-core": "~2.99.1",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~3.0.0",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-jasmine": "~1.1.2",
    "karma-jasmine-html-reporter": "^0.2.2",
    "protractor": "~5.4.0",
    "ts-node": "~7.0.0",
    "tslint": "~5.11.0",
    "typescript": "~3.1.1"
  }
}
```

I have looked at the package.json file and it seems to reference an earlier version of mobX and mobx-angular, like so

```
    "mobx": "^4.15.1",
    "mobx-angular": "3.0.3"
```

But when i try to install the latest version from npm the package.json file shows this?

```
    "mobx": "^5.14.2",
    "mobx-angular": "3.0.3",
```

Is the latest release on npm??

**Comment by maxsoulard** [comment]
Hi,

Having same problem here on IE11.
It seems that the version you are referencing which has mobx@4.15.x will be released in a 8.5.3 of angular-tree-component (based on the versioned package.json on master) but it doesn't exist yet on npm. 
When will this version with mobx downgrade be released on npm?

Thanks.

**Comment by adamkleingit** [comment]
The downgrade was merged.
Please use angular-tree-component 8.5.4
